great_expectations/dataset/pandas_dataset.py

```diff
@@ -84,6 +84,80 @@ def inner_wrapper(self, column, mostly=None, result_format=None, *args, **kwargs
 
         return inner_wrapper
 
+    @classmethod
+    def column_pair_map_expectation(cls, func):
+        """
+        The column_pair_map_expectation decorator handles boilerplate issues surrounding the common pattern of evaluating
+        truthiness of some condition on a per row basis across a pair of columns.
+        """
+
+        @cls.expectation(inspect.getargspec(func)[0][1:])
+        @wraps(func)
+        def inner_wrapper(self, column_A, column_B, mostly=None, ignore_row_if="both_values_are_missing", result_format=None, *args, **kwargs):
+
+            if result_format is None:
```

Comment:
Use `parse_result_format` on the result here.

Reply:
Can I suggest we open that as a new issue? I'm not fully up to speed on `parse_result_format`, and it looks like it's not being applied in `column_map_expectation` either.

Reply:
Sure...although it should, and all of the other decorators already do. I just created a PR for pandas column_map_expectation. It'll be  simple fix to add to this once this PR is in.